src/core_plugins/region_map/public/region_map_vis.js

```diff
@@ -36,6 +36,8 @@ VisTypesRegistryProvider.register(function RegionMapProvider(Private, regionmaps
         wms: config.get('visualization:tileMap:WMSdefaults'),
         mapZoom: 2,
         mapCenter: [0, 0],
+        outlineWeight: 1,
```

Comment:
what was the default outlineWeight before ?

Reply:
1